docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3.3"
+
+services:
+  mariadb-nmo:
+    image: lscr.io/linuxserver/mariadb:latest
+    container_name: mariadb-nmo
+    environment:
+      - PUID=501
+      - PGID=20
+      - MYSQL_ROOT_PASSWORD=password123
+      - MYSQL_USER=root
+      - TZ=America/Los_Angeles
+    volumes:
+      - ./.db:/config
+    ports:
+      - "5100:3306"
+    restart: unless-stopped
+    networks:
+      - network
+
+  baseball:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: baseball
+    depends_on:
+      - mariadb-nmo
+    restart: "no"
```

Comment:
when I ran your file, it initially gave me the error message of: ERROR 2002 (HY000): Can't connect to MySQL server on 'mariadb-nmo' (115)
since you set the baseball container to no restart, it exited out right away. I can manually start it again from docker dashboard and it would run the next step. I'm not sure what's the most correct way to adjust it, but maybe change the restart setting to another option, like unless-stopped for example.